rasa/core/training/story_reader/story_reader.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         template_vars: Optional[Dict] = None,
         use_e2e: bool = False,
         source_name: Text = None,
+        unfold_or_utterances: bool = True,
```

Comment:
this needs some documenation, not sure what this parameter is used for

Reply:
yes, missed that, done.